packages/api/app/auth.js

```diff
@@ -43,7 +43,11 @@ async function ensureAuthorized(req, res, next) {
     const userModel = new User();
     const access = new AccessToken();
 
-    await userModel.get({ userName });
+    // TODO [MHS, 2019-11-18] This might have to change for launchpad auth and
+    // only be done during non-launchpad auth.
+    if (!launchpadProtectedAuth()) {
+      await userModel.get({ userName });
+    }
```

Comment:
Okay so let me know if this is correct. 
You're doing this because the current Launchpad auth we have does not add the user to the user table after we get the username. 
So what happens if that get line is hit when it's launchpad auth?

Reply:
so we're authorizing when we check that the saml response contains the usergroup that cumulus is configured with.  We never store the username anywhere (but the jwt). So when we ensureAuthorization when we have launchpad protecting the api, we don't bother to check if the userName is in the userTable. It won't be hit with Launchpad Auth, that's the point of the if test.